### PR TITLE
ZIOS-8936: Wrongly positioned or missing avatar background in skeleton when switching accounts

### PR DESCRIPTION
**Pull request by nicorsm**
### Issues

1) While switching between accounts, the blur effect of `SkeletonViewController` is different than the one in `BackgroundViewController`.
2) While switching from personal to team account, the `SkeletonViewController` background is grey.

### Causes

1) The `UIVisualEffectView` of `SkeletonViewController` wasn't scaled by 1.4x.
2) The personal avatar was saved inside the `Account` object only on personal accounts, not team accounts. See https://github.com/wireapp/wire-ios-data-model/pull/419 for further information.

### Solutions

1) I've unified the blurring logic in both views.
2) I've added a `teamImageData` field inside the `Account` object.

## Dependencies

Needs releases with:

- [x] https://github.com/wireapp/wire-ios-data-model/pull/419
- [x] https://github.com/wireapp/wire-ios-sync-engine/pull/665

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/SkeletonViewController.swift [hunk 1]
```diff
@@ -254,9 +254,12 @@ class SkeletonViewController: UIViewController {
         super.init(nibName: nil, bundle: nil)
         
         backgroundImageView.contentMode = .scaleAspectFill
-        backgroundImageView.transform = CGAffineTransform(scaleX: 1.4, y: 1.4)
         
-        if let imageData = account.imageData, let image = UIImage(data: imageData) {
+        let factor = BackgroundViewController.backgroundScaleFactor
+        backgroundImageView.transform = CGAffineTransform(scaleX: factor, y: factor)
+        blurEffectView.transform = CGAffineTransform(scaleX: factor, y: factor)
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/SkeletonViewController.swift]: So we are essentially scaling the image two times? Once on the imageView and then again on the effectView.

Comment (nicorsm) [reply Wire-iOS/Sources/UserInterface/SkeletonViewController.swift]: That's the behavior already used in `BackgroundViewController`. I've checked out that you can achieve the same effect by using an image with a size of 40px and scaling only the `UIImageView`. Changes in the next commit 🔜.

File: Wire-iOS/Sources/UserInterface/ConversationList/BackgroundViewController.swift [hunk 2]
```diff
@@ -125,8 +125,9 @@ final public class BackgroundViewController: UIViewController {
         
         self.updateStatusBarBlurStyle()
         
-        self.blurView.transform = CGAffineTransform(scaleX: 1.4, y: 1.4)
-        self.imageView.transform = CGAffineTransform(scaleX: 1.4, y: 1.4)
+        let factor = BackgroundViewController.backgroundScaleFactor
+        //self.blurView.transform = CGAffineTransform(scaleX: factor, y: factor)
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/ConversationList/BackgroundViewController.swift]: Commented out code

**Comment by typfel** [comment]
@nicorsm will you update this PR?

**Comment by nicorsm** [comment]
@typfel Yes, I'm updating dependencies locally right now.

**Pull request merged**